webui/package.json

```diff
@@ -35,15 +36,19 @@
     "react-router-dom": "^6.5.0",
     "react-simple-code-editor": "^0.13.1",
     "react-syntax-highlighter": "^15.5.0",
+    "remark": "^14.0.2",
     "remark-gfm": "^3.0.1",
     "remark-html": "^15.0.1",
+    "unist": "^8.0.11",
```

Comment:
@eladlachmi this line fails `npm install` for me. Couldn't find it on NPM manually either. 

Additionally, I think this PR also introduces a `package-lock.json` file at the root of the repository (i.e. not under `webui/`) by mistake..